tests/fixtures/search.py

```diff
@@ -44,34 +44,26 @@ def __init__(
         self, db: DatabaseTransactionFixture, services: Services, test_id: TestIdFixture
     ):
         self.search_config = SearchTestConfiguration.from_env()
-        self.services_container = services
         self.index_prefix = test_id.id
 
         # Set up our testing search instance in the services container
         self.search_container = Search()
         search_config_dict = self.search_config.model_dump()
         search_config_dict["index_prefix"] = self.index_prefix
         self.search_container.config.from_dict(search_config_dict)
-        self.services_container.search.override(self.search_container)
+        services.search.override(self.search_container)
 
         self.db = db
-        self.client: OpenSearch = services.search.client()
-        self.service: SearchServiceOpensearch1 = services.search.service()
-        self.index: ExternalSearchIndex = services.search.index()
+        self.client: OpenSearch = services.search().client()
+        self.service: SearchServiceOpensearch1 = services.search().service()
+        self.index: ExternalSearchIndex = services.search().index()
         self.revision: SearchSchemaRevision = (
-            services.search.revision_directory().highest()
+            services.search().revision_directory().highest()
         )
 
-        # Make sure the services container is wired up with the newly created search container
-        wire_container(self.services_container)
-
     def close(self):
         # Delete our index prefix
         self.client.indices.delete(f"{self.index_prefix}*")
-
-        # Unwire the services container
-        self.services_container.unwire()
-        self.services_container.search.reset_override()
```

Comment:
The search fixture used to do a bunch of work to wire and unwire the container that is no longer necessary.